python/nano/src/bigdl/nano/deps/openvino/pytorch/metric.py

```diff
@@ -21,7 +21,12 @@
 
 class PytorchOpenVINOMetric(BaseOpenVINOMetric):
     def __init__(self, metric, higher_better=True):
-        invalidInputError(isinstance(metric, Metric),
+        # to support openvino quantization of chronos forecasters
+        if hasattr(metric, '__name__'):
+            forecast_metric = metric.__name__ in ['mae', 'mse', 'rmse', 'mape', 'smape', 'r2']
+        else:
+            forecast_metric = False
+        invalidInputError(isinstance(metric, Metric) or forecast_metric,
```

Comment:
This is a valid tmp change for chronos but may not be a good change for nano.

Issues here is that openvino quantization does not really need a torchmetric for metric, but also support a function with 2 tensor input like shown below.
https://github.com/intel-analytics/BigDL/blob/a896e8520ceb175e4d7a9aa1397757a749c95788/python/nano/src/bigdl/nano/deps/openvino/core/metric.py#L43

**Should we just relax the check in line 26 and only check the signature (2 args).**

BTW, please also change the parameter order in

https://github.com/intel-analytics/BigDL/blob/eba2c223a8142e7a6c6a57c8161669dc327da425/python/chronos/src/bigdl/chronos/forecaster/base_forecaster.py#L2037

This may work for some symmetric metric but errors may happen for some asymmetric metrics.

@MeouSker77 please provide any thoughts why a strict torchmetric check is placed here, I should have forgot something.

Reply:
> @MeouSker77 please provide any thoughts why a strict torchmetric check is placed here, I should have forgot something.

I don't know why, either. And I also think this check is unnecessary.

Reply:
> 

okk, I will try to relax the strict torchmetric check here